### PR TITLE
Ran flake8, black, and tests

### PR DESCRIPTION
**Pull request by intendo**
Just cleaning up some code flake8, black, and making sure tests still pass.

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -14,17 +14,17 @@
 import sys
 
 
-sys.path.insert(0, os.path.abspath('../'))
+sys.path.insert(0, os.path.abspath("../"))
```
Comment (sobolevnrm) [inline docs/conf.py:17]: Why is this needed?

Comment (intendo) [reply docs/conf.py:17]: I did not change the code, black simply changed the quotations marks used. I don't think it should be needed. I was just linting and formatting.

**Review by sobolevnrm — APPROVED** [review]
I mis-read the diff; sorry.
